python/cudf/cudf/tests/test_rank.py

```diff
@@ -12,29 +12,53 @@ def test_series_rank():
     pdf["col2"] = np.array([5, 4, np.nan, 5, 8, 5, np.inf, np.nan, 6, -np.inf])
     gdf = DataFrame.from_pandas(pdf)
 
-    def _check(gs, ps, ascending, method, na_option):
+    def _check(gs, ps, method, na_option, ascending, pct):
         ranked_gs = gs.rank(
-            ascending=ascending, method=method, na_option=na_option)
+            method=method, na_option=na_option, ascending=ascending, pct=pct
+        )
         ranked_ps = ps.rank(
-            ascending=ascending, method=method, na_option=na_option)
+            method=method, na_option=na_option, ascending=ascending, pct=pct
+        )
         if not ranked_ps.equals(ranked_gs.to_pandas()):
-            print(pd.concat(
-                [ps, ranked_ps, gs.to_pandas(), ranked_gs.to_pandas()],
-                axis=1))
+            print(
+                pd.concat(
+                    [ps, ranked_ps, gs.to_pandas(), ranked_gs.to_pandas()],
+                    axis=1,
+                )
+            )
         assert_eq(ranked_ps, ranked_gs.to_pandas())
 
-    # TODO pct [True, False]
     for ascending in [True, False]:
```

Comment:
We should use `pytest.parametrize` to run each of these combinations as an individual test.